modules/javafx.graphics/src/main/java/com/sun/javafx/application/WindowsHighContrastScheme.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+package com.sun.javafx.application;
+
+import com.sun.javafx.PlatformUtil;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Locale;
+import java.util.ResourceBundle;
+
+/**
+ * Enumeration of possible high contrast scheme values.
+ * <p>
+ * For each scheme, a theme key is defined. These keys can be
+ * used in a resource bundle that defines the theme name values
+ * for supported locales.
+ * <p>
+ * The high contrast feature may not be available on all platforms.
+ */
+enum WindowsHighContrastScheme {
```

Comment:
I'd rather keep the old name: `HighContrastScheme`: It is in a non-platform specific package, and it applies to all platforms (even only with `NONE`).
You could add a comment about `HIGH_CONTRAST_*` enum constants being only available on Windows.

Reply:
But it never applied to all platforms, it reflects exactly the Windows implementation of high-contrast schemes. It will be moved once we add style themes.

Reply:
ok